README.md

```diff
@@ -270,6 +275,169 @@ validator -globbing "/path/to/files/**/*.json"
 validator -globbing "/path/*.json" /path/to/search
 ```
 
+## Calling the config-file-validator programatically
+
+The config-file-validator can be called programatically from within a Go program through the `cli` package.
+
+### Default configuration
+
+The default configuration will perform the following actions:
+
+* Search for all supported configuration file types at the cwd
```

Comment:
Maybe this.

```suggestion
* Search for all supported configuration file types in current director and its subdirectories
```

Because `cwd` is a bit obscure